src/p2p/ModeSystemFuncs.ts

```diff
@@ -670,11 +674,32 @@ export function getExpiredRemovedV3(
     )
   }
 
+  const counters = {
+    add,
+    remove,
+    numApoptosizedRemovals: numApoptosizedRemovals,
+    numProblematicRemovals: numProblematicRemovals,
+    numProblematicNodes: problematicNodes.length,
+    canRemoveProblematicNodesThisCycle: canRemoveProblematicNodesThisCycle,
+    numPotentiallyExpiredRemovals: numPotentiallyExpiredRemovals,
+    numExpiredRemovals: numExpiredRemovals,
+  }
   nestedCountersInstance.countEvent(
-      'p2p',
-      `results of getExpiredRemovedV3: numApoptosizedRemovals: ${numApoptosizedRemovals}, numProblematicRemovals: ${numProblematicRemovals}, numExpiredRemovals: ${numExpiredRemovals}, removed: ${remove}`
+    'p2p',
```

Comment:
I dont think there will be too many permutations of this during a nodes 'tour of duty'. .. but we could put this behind a flag too if desireable... but its very useful debugging data.